src/Storages/ObjectStorage/StorageObjectStorage.cpp

```diff
@@ -86,9 +89,25 @@ StorageObjectStorage::StorageObjectStorage(
     , distributed_processing(distributed_processing_)
     , log(getLogger(fmt::format("Storage{}({})", configuration->getEngineName(), table_id_.getFullTableName())))
 {
-    ColumnsDescription columns{columns_};
+    try
+    {
+        configuration->update(object_storage, context);
+    }
+    catch (...)
+    {
+        // If we don't have format or schema yet, we can't ignore failed configuration update, because relevant configuration is crucial for format and schema inference
+        if (mode <= LoadingStrictnessLevel::CREATE || columns_.empty() || (configuration->format == "auto"))
+        {
+            throw;
+        }
+        else
+        {
+            tryLogCurrentException(__PRETTY_FUNCTION__);
```

Comment:
```suggestion
            tryLogCurrentException(log);
```
(better to include table name in this exception, which will be done by using logger instead of pretty function)